src/java/org/apache/cassandra/service/accord/AccordCommandStore.java

```diff
@@ -333,22 +331,32 @@ public AccordStateCache.Instance<Key, CommandsForKey, AccordSafeCommandsForKey>
     {
         return commandsForKeyCache;
     }
-    Command loadCommand(TxnId txnId)
+
+    @Nullable
+    Runnable appendToKeyspace(Command before, Command after)
     {
-        return AccordKeyspace.loadCommand(this, txnId);
+        if (after.keysOrRanges() != null && after.keysOrRanges() instanceof Keys)
+            return NO_OP;
```

Comment:
if we return `null` the cache won't waste time handing to another thread for saving